utils/installer/config_win.example.lua

```diff
@@ -79,14 +78,14 @@ lvim.builtin.terminal.active = false
 -- lvim.builtin.terminal.shell = "pwsh.exe -NoLogo"
 
 -- nvim-tree has some performance issues on windows, see kyazdani42/nvim-tree.lua#549
-lvim.builtin.nvimtree.setup.diagnostics.enable = false
-lvim.builtin.nvimtree.setup.filters.custom = false
-lvim.builtin.nvimtree.setup.git.enable = false
-lvim.builtin.nvimtree.setup.update_cwd = false
-lvim.builtin.nvimtree.setup.update_focused_file.update_cwd = false
+lvim.builtin.nvimtree.setup.diagnostics.enable = nil
+lvim.builtin.nvimtree.setup.filters.custom = nil
```

Comment:
I believe that this is the only one needed

Reply:
to fix the CI issues, go to the repo, do these steps
```sh
git reset HEAD~1 --hard
```
now change only the line with the `lvim.builtin.nvimtree.setup.filters.custom`
you can use whatever commit title you want, as long as it passes https://github.com/LunarVim/LunarVim/blob/rolling/CONTRIBUTING.md#commit-messages
```sh
git add utils/installer/config_win.example.lua
git commit -m "docs(windows): update example config for nvim-tree" 
git push --force -u
```